changelog.d/20230707_184703_ollie_scriv.md

```diff
@@ -0,0 +1,20 @@
+### Changed
+
+- Rel8's API regarding aggregation has changed significantly, and is now a closer match to Opaleye. 
+
+  The previous aggregation API had `aggregate` transform a `Table` from the `Aggregate` context back into the `Expr` context:
+
+  ```haskell
+  myQuery = aggregate do
+    a <- each tableA
+    return $ liftF2 (,) (sum (foo a)) (countDistinct (bar a))
+  ```
+
+  This API seemed convenient, but has some significant shortcomings. The new API requires an explicit `Aggregator` be passed to `aggregate`:
+
+  ```haskell
+  myQuery = aggregate (liftA2 (,) (sumOn foo) (countDistinctOn bar)) do
+    each tableA
+  ```
+
+  For more details, see [#235](https://github.com/circuithub/rel8/pull/235)
```

Comment:
I know you link to the pull request, but I think it is also worth mentioning the difference between `aggregate` and `aggregate1` here. In particular, the behaviour of `aggregate` has changed such that it returns a "zero"/default value when a query returns zero rows, whereas before you would have to wrap it in `optional`. `aggregate1` behaves the same as the previous `aggregate`.

I also the think the addition of support for `FILTER WHERE` is worth highlighting too.

Reply:
I've expanded on this changelog fragment with all points - mind taking another look?